.github/workflows/management-cluster-rosa.yml

```
@@ -48,8 +31,8 @@ env:
 
 jobs:
   management-cluster:
+    if: false  # DISABLED: AWS/ROSA support removed (ARO-27468)
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)